applications/luci-app-upnp/Makefile

```diff
@@ -6,7 +6,7 @@
 
 include $(TOPDIR)/rules.mk
 
-LUCI_TITLE:=Universal Plug & Play configuration module
+LUCI_TITLE:=UPnP IGD & PCP/NAT-PMP configuration module
```

Comment:
What if make it "Universal Plug & Play (UPnP IGD) and NAT-PMP/PCP".
Then it will be easier to find

Reply:
How about both? 
```Makefile
LUCI_TITLE:=Universal Plug & Play (UPnP) IGD & PCP/NAT-PMP configuration module
```